core/src/zeit/content/link/redirect.py

```diff
@@ -61,4 +61,4 @@ def _adjust_workflow(source, target):
         zeit.cms.config.required('zeit.cms', 'live-prefix'),
     )
     log = zeit.objectlog.interfaces.ILog(target)
-    log.log(_('Redirect created for %s', url))
+    log.log(_('Redirect created for {url}', mapping={'url': url}))
```

Comment:
```suggestion
    log.log(_('Redirect created for ${url}', mapping={'url': url}))
```

ist die korrekte gettext Syntax